pkg/export/service.go

```diff
@@ -133,6 +130,18 @@ func (e *exporter) Logger() logger.Logger {
 	return e.logger
 }
 
+func (e *exporter) newRoute(r config.Route) routes.Route {
+	natsTopic := fmt.Sprintf("%s.%s", NatsSub, r.NatsTopic)
+	var route routes.Route
+	switch r.Type {
+	case "mfx":
```

Comment:
Maybe a const for this?

Reply:
I will go into unneccessary mess with naming, how to name that const ? `mfx`? I have already a package `mfx` which I would have to rename than

Reply:
`mfxType`